<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_main_oper">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Операция</string>
    </property>
    <property name="icon">
     <iconset resource="logo.qrc">
      <normaloff>:/logo_for_buttons/operations.png</normaloff>:/logo_for_buttons/operations.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_main_books">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>   Книги</string>
    </property>
    <property name="icon">
     <iconset resource="logo.qrc">
      <normaloff>:/logo_for_buttons/books.png</normaloff>:/logo_for_buttons/books.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_main_clients">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Читатели</string>
    </property>
    <property name="icon">
     <iconset resource="logo.qrc">
      <normaloff>:/logo_for_buttons/clients.png</normaloff>:/logo_for_buttons/clients.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_main_empl">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Сотрудники</string>
    </property>
    <property name="icon">
     <iconset resource="logo.qrc">
      <normaloff>:/logo_for_buttons/employee.png</normaloff>:/logo_for_buttons/employee.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tbl_wdgt_add_user">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>70</y>
      <width>961</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>operation</string>
     </attribute>
     <widget class="QTableWidget" name="operations_tbl_wid">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>71</y>
        <width>961</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>название книги</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>имя клиента</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>тип</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>жанр книги</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>от</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>до</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_record_oper">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledit_title_book">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Введите название книги</string>
      </property>
     </widget>
     <widget class="QComboBox" name="operation_cmb_box">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Выдача</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Возврат</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="days_cmb_box">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Операция</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Кол-во дней</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lidit_name_client">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Введите имя клиента</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>books</string>
     </attribute>
     <widget class="QTabWidget" name="tbl_wdgt_all_books">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Книги</string>
       </attribute>
       <widget class="QTableWidget" name="tbl_wdgt_all_books_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>591</width>
          <height>411</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>id книги</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>описание</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>автор</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="btn_add_book">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>30</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить книгу</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Редакт или удаление</string>
       </attribute>
       <widget class="QLineEdit" name="ledit_search_book">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Введите название</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_title">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_search_title">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>361</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск книги по названию</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_search_book">
        <property name="geometry">
         <rect>
          <x>224</x>
          <y>42</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_inf">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>461</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Если вы ввели корректное название, поля ниже должны были заполнится</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_id_book">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_genre">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>270</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_author">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>320</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="txt_edit_update_desclabelrip">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>321</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_title">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_id_book">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_genre">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Жанр</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_author">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Автор</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_update_book">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_del_book">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>clients</string>
     </attribute>
     <widget class="QTabWidget" name="tbl_wdgt_all_clients">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>951</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Читатели</string>
       </attribute>
       <widget class="QTableWidget" name="tbl_wdgt_all_books_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>591</width>
          <height>411</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>имя</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>номер телефона</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>почта</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="btn_add_client">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>30</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить читателя</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Редакт или удаление</string>
       </attribute>
       <widget class="QLineEdit" name="ledit_search_client">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Введите название</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_name_c">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_search_client_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>361</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск читаиеля по имени</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_search_client">
        <property name="geometry">
         <rect>
          <x>224</x>
          <y>42</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
       <widget class="QLabel" name="lnl_inf_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>461</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Если вы ввели корректное имя, поля ниже должны были заполнится</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_num_c">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_mail_c">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>270</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_name_c">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_num_c">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>номер</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_mail_c">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>почта</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_update_client">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_del_client">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>employee</string>
     </attribute>
     <widget class="QTabWidget" name="tb_wdgt_empl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>951</width>
        <height>441</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Добавить сотруднка</string>
       </attribute>
       <widget class="QLineEdit" name="ledit__add_name_e">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_add_mail_e">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_add_pass_e">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_add_name_e">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_add_mail_e">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_add_pass_e">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_add_e">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Редактировать </string>
       </attribute>
       <widget class="QLabel" name="lbl_update_name_e">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_mail_e">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Почта</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_update_pass_e_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit__update_name_e_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_mail_e">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ledit_update_pass">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_update_e">
        <property name="geometry">
         <rect>
          <x>74</x>
          <y>180</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>логотип</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
